src/gaia/sync.clj

```diff
@@ -23,14 +23,26 @@
   (when (seq if-seq)
     (log/info! message if-seq)))
 
+(defn- guess-owner
+  "Guess a default :owner name from a workflow name like
+  owner_type_timestamp or owner_type_timestamp__description."
+  [workflow]
+  (first (string/split (name workflow) #"_" 2)))
```

Comment:
Seems reasonable, is this format documented anywhere?

Reply:
Good question. I'll document that better in the workflow builder and the Gaia README.

I considered modularizing that code in the workflow builder from its two callers, but deferred for now.

BTW, I have corresponding changes to wcEcoli that we can use once Gaia client 0.0.8 is on pypi and this new Gaia server is on gaia-prime. It sets the workflow property for the sake of the rabbit queue. I'll send a PR soon.

Reply:
I mixed those up. The `__description` suffix is part of the storage root path, not the workflow name.

Reply:
Cool, I have some changes in `gaia` that should go in there too, from the previous PR (currently making sure all feedback is addressed there).

Reply:
Yes. I'm looking forward to pushing these two changes to gaia-prime, pypi, and clojars.
Which Gaia PR do you want to merge first? I'm flexible.

Reply:
Go ahead and merge yours first, and I'll make sure everything is working with all the changes + the new workflow specific sisyphus workers.